features/draft/spec/css-color-hdr-1.yml.dist

```diff
@@ -3,11 +3,9 @@
 
 status:
   baseline: false
-  support:
-    chrome: "135"
-    chrome_android: "135"
+  support: {}
```

Comment:
Claiming no support seems OK since keywords were changed and it's not going to be many milestones supporting the old ones.